config/rbac/role.yaml

```diff
@@ -79,6 +79,21 @@ rules:
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
+metadata:
+  name: manager-role
```

Comment:
there are several roles named "manager-role" in this file.  Please give these roles names that make it clearer what purpose they serve(e.g. cluster-pull-secret-reader").  Otherwise admins are just going to see a "manager-role" role in the "openshift-config" NS and have no idea why it exists/what uses it.

Reply:
these roles are generated by kubebuilder that translate the RBAC settings such as `// +kubebuilder:rbac:groups=core,namespace=openshift-config,resources=secrets,verbs=get;list;watch` to the roles.

Reply:
looks like it's controlled by the makefile, so we can change it:

https://github.com/kubernetes-sigs/kubebuilder/discussions/2297#discussioncomment-3766946

https://github.com/openshift/lightspeed-operator/blob/e76f8bb870bb1564aa33f55873d70e3fbca516b2/Makefile#L102

though i don't see a way to make it unique per role unfortunately.